sdk/eventhub/Azure.Messaging.EventHubs/src/Processor/PartitionPump.cs

```diff
@@ -80,8 +78,8 @@ internal class PartitionPump
         private Task RunningTask { get; set; }
 
         /// <summary>
-        ///   The reason why the partition processor is being closed.  This member is only used in case of failure, as a Shutdown
-        ///   or OwnershipLost close reason will be specified by the event processor.
+        ///   The reason why the associated partition processor is being closed.  This member is only used in case of failure,
+        ///   as a Shutdown or OwnershipLost close reason will be specified by the event processor.
```

Comment:
nit:  I think you're missing "such" here....

Reply:
Maybe my wording was not the best. I meant something like "This member is only used in case of failure, as other kinds of reasons will be specified by the event processor".

Reply:
Changed it to `This member is only used in case of failure. Shutdown and OwnershipLost close reasons will be specified by the event processor.`